style.css

```diff
@@ -0,0 +1,998 @@
+:root {
+  --body-font: "Poppins,sans-serif";
```

Comment:
- [ ] kindly consider using the same font as the one in the provided design to implement well the design 
- the way your font looks 👇 
<img width="281" alt="Screen Shot 2023-03-31 at 15 07 58" src="https://user-images.githubusercontent.com/60146030/229128487-535c07e8-536c-4f56-8985-b1ac5bd47b05.png">

- figma font 👇 
<img width="609" alt="Screen Shot 2023-03-31 at 15 08 53" src="https://user-images.githubusercontent.com/60146030/229129015-a2f2bb5e-2d45-4a51-82c5-ef392656ce93.png">